pyproject.toml

```diff
@@ -80,6 +80,7 @@ dependencies = [
     "tortoise-orm==0.21.6",
     "uvloop~=0.20",
     "web3~=7.2",
+    "aerich~=0.7.2",
```

Comment:
Let's make this dependency optional. A majority of users don't need migrations, and dependency tree is already very bloated. If it's not installed -- just ignore the whole migrations part.